tot/models/models_simple.py

```diff
@@ -204,11 +250,15 @@ def predict(self, df: pd.DataFrame, df_historic: pd.DataFrame = None):
         _check_min_df_len(df=df, min_len=self.n_forecasts)
         if df_historic is not None:
             df = self.maybe_extend_df(df_historic, df)
```

Comment:
Suggestion: pass args by name, not position, to avoid positional mistakes.

Reply:
note: in `maybe_extend_df` and `maybe_drop_added_values_from_df` you could save a line each by not setting, but directly passing `samples = self.n_lags`